### DISCUSSION TITLE
Theme: how to choose "sub theme" version?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#5521] [Q&A] [answered]
Hi,
the cosmo theme has these 3 kinds of predefined styles.

Previously in quarto the default cosmo was the "1", in quarto 1.4 the default is "3".

Is there a way to set cosmo and one of these 3 sub settings?

Thank you

![image](https://github.com/quarto-dev/quarto-cli/assets/30607/5d356740-026f-48b3-864d-c00aa631d239)

**Comment by mcanouil** [comment, marked as answer]
It's not a "mode" you can switch.
You need to use `scss`, see <https://quarto.org/docs/output-formats/html-themes-more.html> and <https://quarto.org/docs/output-formats/html-themes.html>.

~~At the moment, I don't know how to reuse `$secondary`, `$dark`, _etc._ from `cosmo` theme.~~
Edit: you can't.

Alternatively, if you need less control, you can use `background`, `foreground`, _etc._, yaml keys from `navbar`, see <https://quarto.org/docs/reference/projects/websites.html#navbar>.
So you can set `background` to `primary`, `light`, `dark`, _etc._
Note that this will only change the navbar, nothing else in the theme.

**Reply by aborruso** [reply]
Thank you
